secantTests/TransactionListTests/TransactionStateTests.swift

```diff
@@ -349,43 +349,43 @@ final class TransactionStateTests: XCTestCase {
     
     // MARK: - Amounts
 
-    func testCollapsedPrimaryAmountRoundingForSend() throws {
-        let transaction = TransactionState(
-            memos: [try! Memo(string: "Hi, pay me and I'll pay you")],
-            minedHeight: BlockHeight(1),
-            zAddress: "tmP3uLtGx5GPddkq8a6ddmXhqJJ3vy6tpTE",
-            fee: Zatoshi(10_000),
-            id: "t1vergg5jkp4wy8sqfasw6s5zkdpnxvfxlxh35uuc3me7dp596y2r05t6dv9htwe3pf8ksrfr8ksca2lskzja",
-            status: .paid,
-            timestamp: 1699290621,
-            zecAmount: Zatoshi(25_793_456),
-            isSentTransaction: false,
-            isAddressExpanded: false,
-            isExpanded: false,
-            isIdExpanded: false
-        )
-        
-        XCTAssertEqual(transaction.roundedAmountString, "-0.258")
-    }
-    
-    func testCollapsedPrimaryAmountRoundingForReceive() throws {
-        let transaction = TransactionState(
-            memos: [try! Memo(string: "Hi, pay me and I'll pay you")],
-            minedHeight: BlockHeight(1),
-            zAddress: "tmP3uLtGx5GPddkq8a6ddmXhqJJ3vy6tpTE",
-            fee: Zatoshi(10_000),
-            id: "t1vergg5jkp4wy8sqfasw6s5zkdpnxvfxlxh35uuc3me7dp596y2r05t6dv9htwe3pf8ksrfr8ksca2lskzja",
-            status: .received,
-            timestamp: 1699290621,
-            zecAmount: Zatoshi(25_793_456),
-            isSentTransaction: false,
-            isAddressExpanded: false,
-            isExpanded: false,
-            isIdExpanded: false
-        )
-        
-        XCTAssertEqual(transaction.roundedAmountString, "+0.258")
-    }
+//    func testCollapsedPrimaryAmountRoundingForSend() throws {
```

Comment:
Should this test still work, or should it be deleted?

Reply:
good catch, removed